t/01-basics.t

```diff
@@ -2,11 +2,7 @@ use v6;
 
 use Test;
 
-plan 25;
-
-my constant $root = $?FILE.IO.parent.parent;
```

Comment:
On my system (Debian) these didn't do anything helpful, don't know if that is generally the case, but now it runs.